cpp/change-notes/2021-08-23-getPrimaryQlClasses.md

```diff
@@ -0,0 +1,2 @@
+lgtm,codescanning
+* Added `ElementBase.getPrimaryQlClasses()` predicate, which gets a comma-separated list of the names of the primary CodeQL classes to which this element belongs.
```

Comment:
```suggestion
* Added `Element.getPrimaryQlClasses()` predicate, which gets a comma-separated list of the names of the primary CodeQL classes to which this element belongs.
```

I think of `ElementBase` as an internal class that's there for implementation reasons. We can't quite express that in QL, but we can at least express it in documentation. I'm not actually sure why `getAPrimaryQlClass` is defined on `ElementBase` rather than `Element`, but that's not for this PR to change.